GovUk.Frontend.Umbraco.ExampleApp/uSync/v9/ContentTypes/govukfileuploadsettings.config

```diff
@@ -1,97 +1,81 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <ContentType Key="d10b177e-98b9-42c3-a7f6-632c885c6bcf" Alias="govukFileUploadSettings" Level="2">
-	<Info>
-		<Name>File upload settings</Name>
-		<Icon>icon-cloud-upload color-black</Icon>
-		<Thumbnail>folder.png</Thumbnail>
-		<Description>Settings for a file upload component. Ask users to upload something only if it’s critical to the delivery of your service.</Description>
-		<AllowAtRoot>False</AllowAtRoot>
-		<IsListView>False</IsListView>
-		<Variations>Nothing</Variations>
-		<IsElement>true</IsElement>
-		<HistoryCleanup>
-			<PreventCleanup>False</PreventCleanup>
-			<KeepAllVersionsNewerThanDays></KeepAllVersionsNewerThanDays>
-			<KeepLatestVersionPerDayForDays></KeepLatestVersionPerDayForDays>
-		</HistoryCleanup>
-		<Folder>GOV.UK</Folder>
-		<Compositions>
-			<Composition Key="bf69e8d6-97a6-4b50-bec7-c5993e120339">govukCssClasses</Composition>
-			<Composition Key="37cc75ea-be5c-4107-b9b8-0ef34f4b7c5e">govukErrorMessagePrefix</Composition>
-			<Composition Key="d508995b-a958-4273-8ab3-50735220a1b2">govukGrid</Composition>
-			<Composition Key="d7016563-0b6f-4c3a-b75d-b0bbe66419fd">govukGridColumnClasses</Composition>
-			<Composition Key="2bc3c434-f3ff-495d-b360-1a005245602e">govukLabelIsPageHeading</Composition>
-			<Composition Key="23e1a03d-dc42-4318-9e17-d458dde2dba6">govukModelProperty</Composition>
-			<Composition Key="f2365c7e-08c3-4ef9-abbc-e337e34de272">govukValidationRequired</Composition>
-		</Compositions>
-		<DefaultTemplate></DefaultTemplate>
-		<AllowedTemplates />
-	</Info>
-	<Structure />
-	<GenericProperties>
-		<GenericProperty>
-			<Key>cac445e1-9d87-452e-94de-a70e43ff09ff</Key>
-			<Name>Allowed file types</Name>
-			<Alias>errorMessageAllowedFileExtensions</Alias>
```

Comment:
Moved the extension validation to the `AllowedFileTypesAttribute`

The reason why `AllowedFileExtensionsAttribute` is marked as Obsolete is because I don't know where it is used and do not want to create a breaking change. Marking is Obsolete allows people to migrate in an orderly fashion and we can delete it in a few months or so when necessary.

Reply:
Yes, I understand your thinking on the use of `[Obsolete]`. 

However, `ApplicationBuilderExtensions.cs` connects the obsolete `AllowedFileExtensionsAttribute`  to a property with the alias `errorMessageAllowedFileExtensions`, so by removing that property you've broken it.

In this case I would recommend removing it because it was only added in v6.0.0-alpha004 and we haven't published v6.0.0 yet. It's the nature of alphas that features come and go, and projects use them at their own risk.

Reply:
Removed `[AllowedFileExtensions]`